main.go

```diff
@@ -473,19 +473,25 @@ func getMaxCIDRMask(addr net.IP) net.IPMask {
 
 func safeAddInterfaceAddress(swIfIndex uint32, addr *net.IPNet) (err error) {
 	maskSize, _ := addr.Mask.Size()
-	if vpplink.IsIP6(addr.IP) && maskSize != 128 {
-		err = vpp.AddInterfaceAddress(swIfIndex, &net.IPNet{
-			IP:   addr.IP,
-			Mask: getMaxCIDRMask(addr.IP),
-		})
+	if vpplink.IsIP6(addr.IP) {
+		err = vpp.EnableInterfaceIP6(swIfIndex)
```

Comment:
Ach, yep forgot this. But https://github.com/calico-vpp/vpp-manager/pull/14/files#diff-7ddfb3e035b42cd70649cc33393fe32cR649 should fix it better nope ?

Reply:
Yep, removed this hunk from this PR.

Reply:
Cool thx 👍